### PR TITLE
Fix GS44 - Meta.later_add is not a function

### PR DESCRIPTION
**Pull request by G-dH**
The fix addresses compatibility with the GS 44 and is backwards compatible.

### CODE REVIEW

**Comment by Newbytee** [comment]
@RensAlthuis did you see this PR?

**Comment by hschimpf** [comment]
It works!

Tested on Manjaro with x11 and GNOME Shell 44.1

@G-dH, can you also add this change to the PR?
`metadata.json`
```diff
  "shell-version": [
    "42",
-   "43"
+   "43",
+   "44"
  ],
```

**Comment by jaxwilko** [comment]
Tested on Debian trixie/sid, GNOME Shell 44.3, can confirm this works and fixes workspace preview

File: workspaceThumbnail.js [hunk 1]
```diff
@@ -190,9 +190,16 @@ var ThumbnailsBoxOverride = {
...
+            if (Meta.later_add) {
+                Meta.later_add(Meta.LaterType.BEFORE_REDRAW, () => {
+                    this._dropPlaceholder.hide();
+                });
+            } else {
+                const laters = global.compositor.get_laters();
+                laters.add(Meta.LaterType.BEFORE_REDRAW, () => {
+                    this._dropPlaceholder.hide();
+                });
+            }
```
Comment (torrocus) [inline workspaceThumbnail.js:202]: Based on [GJS](https://gjs.guide/extensions/upgrading/gnome-shell-44.html#meta-later-add-and-meta-later-remove) [documentation](https://gjs-docs.gnome.org/meta12~12/meta.laters) we can use the new version of function:

> `Meta.later_add` and `Meta.later_remove` are now `Meta.Laters.add` and `Meta.Laters.remove`, respectively. 

IMO something like this should work too:

```suggestion
            const later_function = Meta.later_add || Meta.Laters.add;
            later_function.add(Meta.LaterType.BEFORE_REDRAW, () => {
                this._dropPlaceholder.hide();
            });
```

Comment (G-dH) [reply workspaceThumbnail.js:202]: There is a reason for using `global.compositor.get_laters()`, `Meta.Laters` is a class that needs an instance.

**Review by christophermarklee — APPROVED** [review]

**Pull request closed without merging**